tests/datapipeline/test_datapipeline.py

```diff
@@ -0,0 +1,101 @@
+"""Verify AWS Data Pipeline Creation."""
+from unittest import mock
+import pytest
+
+import copy
+import awscli.customizations.datapipeline.translator
+from foremast.datapipeline.datapipeline import AWSDataPipeline
+
+GOOD_DEF = 	{
+        "objects": [],
+        "parameters": [],
+        "values": {}
+        }
+
+BAD_DEF = 	{
+        "no_object": [],
+        "parameters": [],
+        "values": {}
+        }
+
+TEST_PROPERTIES = {
+    'test_env': {
+        'datapipeline': {
+            'name': 'Test Pipeline',
+            'description': "this is a test",
+            'json_definition': GOOD_DEF
+        }
+    }
+}
+
+@mock.patch('foremast.datapipeline.datapipeline.boto3.Session.client')
+@mock.patch('foremast.datapipeline.datapipeline.get_details')
+@mock.patch('foremast.datapipeline.datapipeline.get_properties')
+def test_create_datapipeline(mock_get_properties, mock_get_details, mock_boto3):
+    """Check data pipeline creation"""
+    generated = {"project": "test"}
+    properties = copy.deepcopy(TEST_PROPERTIES)
+    mock_get_details.return_value.data = generated
+    mock_get_properties.return_value = properties
+    mock_boto3.return_value.create_pipeline.return_value = {}
```

Comment:
`create_pipeline()` should return what AWS would return, https://boto3.readthedocs.io/en/latest/reference/services/datapipeline.html#DataPipeline.Client.create_pipeline:
```python
{
    'pipelineId': 'string'
}
```
And we should assert that this was set properly as `AWSDataPipeline.pipeline_id`.